docs/book/src/migration/multi-group.md

```diff
@@ -29,36 +28,29 @@ can check `api/v1/groupversion_info.go` to find that out:
 package v1
 ```
 
-Then, we'll rename `api` to `apis` to be more clear, and we'll move our
-existing APIs into a new subdirectory, "batch":
-
-```bash
-mv api apis
-mkdir apis/batch
-mv apis/v* batch/
-```
-
-Next, we'll need to update all the references to the old package name. For
-CronJob, that'll be `main.go` and `controllers/cronjob_controller.go`.  If
-you've added additional files to your project, you'll need to track down
-imports there as well.
-
-Finally, we'll add a new line to `PROJECT` that marks this as
-a multi-group project:
-
+When the command `kubebuilder multigroup` be executed it will add a new line 
+to `PROJECT` that marks this a multi-group project:
+                                                      
 ```yaml
 version: "2"
 domain: tutorial.kubebuilder.io
 repo: tutorial.kubebuilder.io/project
 multigroup: true
 ```
 
-While this option doesn't do anything right now, it'll indicate to
-KubeBuilder in future versions that this is a multi-group project.
+Note that this option indicates to KubeBuilder that this is a multi-group project. 
+
+In this way, if the project is not new and has previous APIs already implemented will be
+in the previous structure. See that with the `multigroup`the Kind API's files are
+created under `apis/<group>/<version>` instead of `api/<version>`. So, will be required to manually
+move all for the new structure and then, updated its references.
+
+Then, we'll need to fix all the references to the old package name. For
+CronJob, that'll be `main.go` and `controllers/cronjob_controller.go`.  If
+you've added additional files to your project, you'll need to track down
+imports there as well.
 
-Now that we've migrated this to a multi-group project, normal KubeBuilder
-scaffolding won't work (as of KubeBuilder v2.0.0 -- see the warning at the
-top of this page).  You'll need to copy the setup that KubeBuilder
+You'll need to copy the setup that KubeBuilder
 normally does to add new groups for the moment:
```

Comment:
This sentence does not apply anymore. It used to say that, after enabling multigroup, kubebuilder will still deploy in  `api/VERSION/...` and you had to move them manually to `apis/GROUP/VERSION/...` but this PR already does that. For previous files, it has already been explained how they should be moved, this is talking about future files.